### PR TITLE
Remove unused npm `audit` script from generated packages

### PR DESCRIPTION
**Pull request by jeremymeng**
- as `audit` is not used any more in the azsdk js repo
- and remove unused rush commands. They were copied from azure-sdk-for-js repo but not used any more.
- update test baselines

### CODE REVIEW

**Comment by jeremymeng** [comment]
interesting. Nothing changes from last week but now CI is failing

File: common/config/rush/command-line.json [hunk 1]
```diff
@@ -164,65 +158,36 @@
...
         "summary": "Run typedoc on projects and generate documentation",
         "enableParallelism": true,
         "ignoreMissingScript": true
       },
-  
+
       // Global Commands
-      {
-        "commandKind": "global",
-        "name": "list-files",
```
Comment (qiaozha) [inline common/config/rush/command-line.json:179]: I guess these are copied from somewhere but actually doesn't work at all ?

**Review by qiaozha — APPROVED** [review]
LGTM

File: common/config/rush/command-line.json (hunk 1 above)
Comment (jeremymeng) [reply common/config/rush/command-line.json:179]: yeah looks copied from azsdkjs repo. It's not used anywhere as fare as I know.

**Pull request merged**